README.md

```diff
@@ -31,27 +31,27 @@ The above code is likely to be familiar to any student who has competed in one o
 
 ```python
 from j5.boards import BoardGroup
-from j5.backends.hw import HardwareEnvironment
 
 from j5.boards.sr.v4 import PowerBoard, MotorBoard, ServoBoard, Ruggeduino
-
+from j5.backends.hardware.sr.v4 import (
+    SRV4MotorBoardHardwareBackend,
+    SRV4PowerBoardHardwareBackend,
+    SRV4RuggeduinoHardwareBackend,
+    SRV4ServoBoardHardwareBackend,
+)
 
 class Robot:
 
     def __init__(self):
+        self.power_board = PowerBoard(SRV4PowerBoardHardwareBackend)
```

Comment:
`PowerBoard()` should take an *instance* of `SRV4PowerBoardHardwareBackend`, and also a serial number.

```python
power_boards = SRV4PowerBoardHardwareBackend.discover()
if len(power_boards) == 1:
  self.power_board = list(power_boards)[0]
else:
  raise SomeException
```

but at this point you might as well use `BoardGroup.singular`.

Reply:
a21fe15